packages/server/lib/util/ci_provider.js

```diff
@@ -341,6 +342,16 @@ const _providerCiParams = () => {
       'TRAVIS_PULL_REQUEST_SHA',
     ]),
     wercker: null,
+    // https://docs.netlify.com/configure-builds/environment-variables/#deploy-urls-and-metadata
+    netlify: extract([
+      'BUILD_ID',
+      'CONTEXT',
+      'REPOSITORY_URL',
+      'URL',
+      'DEPLOY_URL',
+      'DEPLOY_PRIME_URL',
+      'DEPLOY_ID',
```

Comment:
I don't totally understand why we're capturing all of the deployment info, what are we doing with this info?

Reply:
maybe nothing yet, maybe show in the Dashboard later (like "view preview URL")